rasa/cli/x.py

```diff
@@ -346,6 +347,14 @@ def run_in_production(args: argparse.Namespace):
     _rasa_service(args, endpoints, None, credentials_path)
 
 
+def _get_config_path(args: argparse.Namespace,) -> Optional[Text]:
+    config_path = cli_utils.get_validated_path(
+        args.config, "config", DEFAULT_CONFIG_PATH
+    )
+
+    return config_path
```

Comment:
I wonder whether we should pass this `validated` config path into Rasa X so we don't need these lines of codes in Rasa X 

```python
if config_path:
        config.default_config_path = config_path
else:
        config.default_config_path = LOCAL_CONFIG_PATH
```
but could just do
```python
config.default_config_path = config_path
```

Reply:
what do you think?

Reply:
Yes, I like that! 
I have implemented it & will show up in next push.

Should we do something similar for these ones that are defined in rasax/community/local.py:
LOCAL_DATA_DIR = "data"
LOCAL_DEFAULT_NLU_FILENAME = "nlu.md"
LOCAL_DEFAULT_STORIES_FILENAME = "stories.md"
LOCAL_DOMAIN_PATH = "domain.yml"
LOCAL_MODELS_DIR = "models"

Probably as part of a new issue...